src/coreclr/System.Private.CoreLib/src/System/Reflection/AssemblyName.CoreCLR.cs

```diff
@@ -106,41 +106,46 @@ internal AssemblyNameFlags RawFlags
         internal void SetProcArchIndex(PortableExecutableKinds pek, ImageFileMachine ifm)
         {
 #pragma warning disable SYSLIB0037 // AssemblyName.ProcessorArchitecture is obsolete
-            ProcessorArchitecture = CalculateProcArch(pek, ifm, _flags);
+            ProcessorArchitecture = CalculateProcArchIndex(pek, ifm, _flags);
 #pragma warning restore SYSLIB0037
         }
 
-        private static ProcessorArchitecture CalculateProcArch(PortableExecutableKinds pek, ImageFileMachine ifm, AssemblyNameFlags aFlags)
+        private static ProcessorArchitecture CalculateProcArchIndex(PortableExecutableKinds pek, ImageFileMachine ifm, AssemblyNameFlags flags)
         {
```

Comment:
I think we can keep this part as is. It is simpler, it has better comments, and it is functional equivalent to the old code.

Reply:
> Ideally, we would delete this and just return ProcessorArchitecture.None to be in sync with the S.R.M

Actually, since we are breaking this one way or another, maybe we should break this all the way and just return `None` ?
There is some time in 8.0 to see if this gets anyone in trouble.